src/style.css

```diff
@@ -0,0 +1,106 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+@import "./rating.css";
+
+:root {
+  --main-accent: #060b35;
+  --blue-accent: #00acdc;
+  --secondary: #616070;
+  --line-height-big: 18px;
+  --line-height-small: 14px;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.product-card {
+  font-family: "Roboto", sans-serif;
+  font-weight: normal;
+  color: var(--main-accent);
+
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: var(#fff);
+}
+
+.product-card__product-img {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__product-name {
+  font-size: 12px;
+  line-height: var(--line-height-big);
+  font-weight: 500;
+  margin-bottom: 4px;
+}
+
+.product-card__product-number {
```

Comment:
it helps a lot to have variables and classes with descriptive names. there can be many numbers on a card and each with a different purpose. would make more sense to name it `product-id` or something